timer.js

```diff
@@ -0,0 +1,14 @@
+let timeLeft = 30;
+
+function timer() {
+    setTimeout(() => {
+        if (timeLeft > 0) {
+            timer();
+        } else {
+            return;
+        }
+
+        timeLeft--;
+        document.querySelector('#game_timer').textContent = `Timer: ${timeLeft}`;
```

Comment:
this line, and line 35 in main.js are identical except for the time value, could this be a function with the time as a param? maybe `displayTimeLeft` ?